### PR TITLE
create dependencyManagement

### PR DESCRIPTION
**Pull request by NadezhdaPrilepa**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,17 +11,18 @@
...
             <artifactId>junit</artifactId>
-            <version>4.10</version>
-            <scope>test</scope>
+            <version>4.12</version>
         </dependency>
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>18.0</version>
+            <version>17.0</version>
```
Comment (vbogdanovsky) [inline pom.xml]: Nadezhda, why did you decrease the version for guava?

File: teach/pom.xml [hunk 2]
```diff
@@ -9,7 +9,16 @@
         <version>1.0-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
-    <groupId>skypeTeachNP</groupId>
     <artifactId>teach</artifactId>
     <packaging>jar</packaging>
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
```
Comment (vbogdanovsky) [inline teach/pom.xml:17]: please add test-scope for junit

Comment (NadezhdaPrilepa) [inline teach/pom.xml:17]: done

**Comment by NadezhdaPrilepa** [comment]
All comments were process

**Comment by vbogdanovsky** [comment]
:+1: OK

**Pull request merged**